stan/math/rev/mat/fun/cov_exp_quad.hpp

```diff
@@ -0,0 +1,295 @@
+#ifndef STAN_MATH_REV_MAT_FUN_COV_EXP_QUAD_HPP
+#define STAN_MATH_REV_MAT_FUN_COV_EXP_QUAD_HPP
+
+#include <stan/math/rev/core.hpp>
+#include <stan/math/prim/arr/meta/is_vector.hpp>
```

Comment:
Can we clean up a lot of these includes? Anything that's not directly used shouldn't be included.